build.sh

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env sh
-set -xeuo pipefail
+#!/usr/bin/env bash
```

Comment:
Why not just `/bin/bash`?

Reply:
Habit ... `/bin/bash` breaks on Mac, so this version works everywhere. My fingers remember it now.